RapidHouseNumbers.js

```diff
@@ -478,8 +461,8 @@
           className => !node.classList.contains(className),
         ) === -1
       ) {
-        const { display } = window.getComputedStyle(node);
-        const visible = display !== "none";
+        const display = node.attributeStyleMap.get("display");
```

Comment:
attributeStyleMap doesnt work in firefox!
![grafik](https://github.com/user-attachments/assets/9884b997-21d4-4617-a59c-a2b21c602bd8)

Reply:
I adapted the visibility check to work on both Chrome and Firefox